evalai/utils/submissions.py

```diff
@@ -0,0 +1,103 @@
+import requests
+import sys
+
+from click import echo, style
+from datetime import datetime
+from dateutil import tz
+
+from evalai.utils.auth import get_request_header
+from evalai.utils.config import API_HOST_URL, EVALAI_ERROR_CODES
+from evalai.utils.urls import URLS
+from evalai.utils.common import validate_token
+
+
+def make_submission(challenge_id, phase_id, file):
+    """
+    Function to submit a file to a challenge
+    """
+    url = "{}{}".format(API_HOST_URL, URLS.make_submission.value)
+    url = url.format(challenge_id, phase_id)
+
+    headers = get_request_header()
+    file_data = {'input_file': file}
+    data = {
+            'status': 'submitting',
+           }
+
+    try:
+        response = requests.post(
+                                url,
+                                headers=headers,
+                                files=file_data,
+                                data=data,
+                                )
+        response.raise_for_status()
+    except requests.exceptions.HTTPError as err:
+        if (response.status_code in EVALAI_ERROR_CODES):
+            validate_token(response.json())
+            echo(style("Error: {}".format(response.json()["error"]), fg="red", bold=True))
+        else:
+            echo(err)
+        sys.exit(1)
+    except requests.exceptions.RequestException as err:
+        echo(err)
+        sys.exit(1)
+    response = response.json()
+    response_message = ("\nYour file {} with the ID {} was successfully submitted.\n\n"
+                        "You can use `evalai submission {}` to view this submission's status.\n").format(file.name,
```

Comment:
Please change its colour
`You can use evalai submission {} to view above submission's status.` to white.